velox/functions/sparksql/tests/JsonObjectKeysTest.cpp

```diff
@@ -40,6 +40,7 @@ TEST_F(JsonObjectKeysTest, basic) {
   expected = makeNullableArrayVector<std::string>({std::nullopt});
   assertEqualVectors(jsonObjectKeys(R"(1)"), expected);
   assertEqualVectors(jsonObjectKeys(R"("hello")"), expected);
+  assertEqualVectors(jsonObjectKeys(R"(invalid json)"), expected);
```

Comment:
Would you please update the document as well? It already mentions the invalid json string, while the example seems not being covered.

https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/spark/json.rst?plain=1#L79-L80

Reply:
Yes, add new case in this doc